modules/loki-deployment-sizing.adoc

```diff
@@ -1,41 +1,77 @@
 // Module is included in the following assemblies:
-//cluster-logging-loki.adoc
+// * logging/cluster-logging-loki.adoc
+// * network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="loki-deployment-sizing_{context}"]
-= Deployment Sizing
-
-Sizing for Loki follows the format of `N<x>._<size>_` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
-
-[NOTE]
-====
-1x.extra-small is for demo purposes only, and is not supported.
-====
-
-.Loki Sizing
-[options="header"]
-|========================================================================================
-|                              | 1x.extra-small  | 1x.small            | 1x.medium
-| *Data transfer*              | Demo use only.  | 500GB/day           | 2TB/day
-| *Queries per second (QPS)*   | Demo use only.  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
-| *Replication factor*         | None            | 2                   | 3
-| *Total CPU requests*         | 5 vCPUs         | 36 vCPUs            | 54 vCPUs
-| *Total Memory requests*      | 7.5Gi           | 63Gi                | 139Gi
-| *Total Disk requests*        | 150Gi           | 300Gi               | 450Gi
-|========================================================================================
-
-[id="CRD-API-support_{context}"]
-== Supported API Custom Resource Definitions
-LokiStack development is ongoing, not all APIs are supported currently supported.
-
-[options="header"]
-|=====================================================================
-| CustomResourceDefinition (CRD)| ApiVersion           | Support state
-| LokiStack      | lokistack.loki.grafana.com/v1       | Supported in 5.5
-| RulerConfig    | rulerconfig.loki.grafana/v1beta1    | Technology Preview
-| AlertingRule   | alertingrule.loki.grafana/v1beta1   | Technology Preview
-| RecordingRule  | recordingrule.loki.grafana/v1beta1  | Technology Preview
-|=====================================================================
-
-:FeatureName: Usage of `RulerConfig`, `AlertingRule` and `RecordingRule` custom resource definitions (CRDs).
-include::snippets/technology-preview.adoc[]
+= Loki deployment sizing
+
+Sizing for Loki follows the format of `<N>x.<size>` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
+
+ifeval::["{context}" == "cluster-logging-loki"]
+:restricted:
+endif::[]
+
```

Comment:
While this `ifeval` should work fine, I have never seen one set in the assembly proper. I suggest moving it above the assembly metadata.